src/hammer-vlsi/technology/nangate45/defaults.yml

```diff
@@ -0,0 +1,61 @@
+# Settings for the nangate45 technology to be overriden by the project.
+technology.nangate45:
+
+# Set some defaults for this technology.
+# For quick performance evaluations, use the typical voltage+temperature corners.
+# Nominal voltage for 7nm technology is 0.70 V and is the only voltage provided
```

Comment:
This should be updated as 45nm and 1.1 V(based on below)?

Reply:
done